docs/adr/0006-pipline-runner.md

```diff
@@ -0,0 +1,33 @@
+# 6) Pipeline runner {#adr_0006}
+
+Date: 2020-12-03
+
+## Status
+
+proposed
+
+## Context
+
+A decision needs to be made on where the pipeline should be executed. The initial prototype has been built using [Github] Actions for it's ease of use and made it possible to quickly set up a repository and full access to a framework that allows us to execute our build steps, testing and publish artifacts. To this day very few technical downsides has been identified by using github to run the pipeline. However there are also strong political arguments against using github since their terms of service conflicts with the goals of the foundations, as github is currently not available to everyone based on [sanctions] enforced by U.S. trade restrictions.
```

Comment:
the point about Github  being restricted by US trade sanctions etc is valid, and indeed something I find not cool - as a person. I am not not an expert on international trade regulations so bear with me, but wouldn't US trade restrictions also apply to any WMF provide infrastructure/service, as WMF is also US-based organisation?
The point is absolutely valid, but this unfair treatment seems more related to US policies/global politics, than to a particular vendor in this context.

Reply:
Yeah I probably should not have brought this up.

Reply:
commit message made me lol though.